### PR TITLE
Issue260 rom

### LINKED ISSUE #78: Clean up of low order models
- [ ]  delete unnecessary models
  - EqAirTempSimple
  - ReducedOrderModelStar
- [x] delete name affix EBCMod
- [ ] rename the parameters to be uniform
  - EqAirTemp package
  - ReducedOrderModel package
  - SolarRadWeightedSum
  - ThermalZonePhysics
  - ThermalZone
  - ZoneBaseRecord
- [x] make ReducedOrderModel replaceable, implement choice in higher level models and change default reduced order model in ThermalZonePhysics to the one with decoupled RWin
- [ ] clean up ZoneBaseRecord
  - No default values
  - clean up white spaces
  - delete old, unused parameter
- [x] implement choice between CorrG and NoCorrG for higher level models

### LINKED ISSUE #151: Check if Record and used model are compatible
We are already raising some warning if the weightfactors are not compatible to the used model. However it might make sense to integrate a mechanism that assert the weightfactors (or other parameters) and if they don't fit we shouldn't simulate the model.

This causes a lot of confusion

### LINKED ISSUE #260: Adopt Annex60.ThermalZones.ReducedOrder as core for AixLib.Buildings.LOM
This will replace `AixLib.Buildings.LOM` by `Annex60.ThermalZones.ReducedOrder`. This will change fundamentals as: 
- handling and calculation of solar radiation
- use of weather bus
- new base record definitions
- new parameters
- new parameter names

### PR DESCRIPTION
**Pull request by mlauster**
This introduces the all-new reduced order models, thermal zone models and multizone models.
This closes #260 
This closes #78 
This closes #151

### CODE REVIEW

File: AixLib/DataBase/ThermalZones/OfficePassiveHouse/package.mo [hunk 1]
```diff
@@ -0,0 +1,5 @@
+within AixLib.DataBase.ThermalZones;
+package OfficePassiveHouse "Contains reocrds for an office as passive house"
```
Comment (marcusfuchs) [inline AixLib/DataBase/ThermalZones/OfficePassiveHouse/package.mo]: @mlauster 
"reocrds" should be "records"

**Comment by mlauster** [comment]
@marcusfuchs, I corrected the typo and added a User's Guide to `Buildings` package. The plan is to integrate `Buildings` into `ThermalZones` on the mid term, see #276. To have it more strict, I deleted the remaining example and the old VDI 6007 validation in `LOM`.
Regarding the order of the VDI 6007 validation cases, they are developed within the Annex60-library and the merging script leads to this kind of order. Let's see if we change it in the Annex60 library.

**Comment by mlauster** [comment]
@marcusfuchs, all conflicts are solved, branch is ready to merge

**Comment by PMehrfeld** [comment]
There is still a branch [issue260_ROM_eqAirTemp](https://github.com/RWTH-EBC/AixLib/tree/issue260_ROM_eqAirTemp).
#260 is already closed.
This PR only merged the branch `issue260_ROM`.
In #260 there is a reference to the still opened issue #276. 

If the branch [issue260_ROM_eqAirTemp](https://github.com/RWTH-EBC/AixLib/tree/issue260_ROM_eqAirTemp) obsolete, please delete it @mlauster.

**Pull request merged**